exodus_lambda/functions/origin_request.py

```diff
@@ -8,9 +9,11 @@
 
 from .base import LambdaBase
 
+CONF_FILE = os.environ.get("EXODUS_LAMBDA_CONF_FILE") or "lambda_config.json"
```

Comment:
```suggestion
CONF_FILE = os.environ.get("EXODUS_LAMBDA_CONF_FILE", "lambda_config.json")
```

Reply:
In general I prefer the former style as Tom has written it now.

The reason is that it allows you to set an empty string for the env var in order to mean "use the default behavior" (while with your suggestion, setting an empty string would just make it crash).

This is useful because sometimes, when various systems are plugged together, you might find yourself in a situation where you can't possibly unset EXODUS_LAMBDA_CONF_FILE, but you *can* set it to an empty string.

For instance, imagine it's coming from an ansible variable filled in when you run a playbook, and there's a default value in inventory already.  If you run ansible at the command-line, you can then do `-e exodus_lambda_conf_file=""` to set a variable to an empty string in order to mean "just use the default".  AFAIK you can't *unset* a variable which has already been set though.

Therefore as a general rule I think it's a good idea to treat empty values the same as absent values when reading config, because sometimes you have a way to make a value empty, but you don't have a way to make it absent.